docs/getting_started/installation.rst

```diff
@@ -11,13 +11,7 @@ Installation
 Development Install
 ===================
 
-The `gos source repository`_ is available on GitHub. Once you have cloned the
-repository and installed all the above dependencies, run the following command
-from the root of the repository to install the main version of **gos**:
-
-.. code-block:: bash
-
-    $ pip install -e '.[dev]'
+The `gos source repository`_ is available on GitHub. Please read its [README.md](https://github.com/gosling-lang/gos?tab=readme-ov-file#development) for development guidelines.
```

Comment:
I noticed that the doc still mentions `pip` while we use `uv`. I just pointed to the `README.md` of the GitHub repo.

Reply:
ah great, thanks!